lib/matplotlib/axes/_axes.py

```diff
@@ -8100,15 +8100,17 @@ def violin(self, vpstats, positions=None, vert=True, widths=0.5,
           - ``cquantiles``: A `~.collections.LineCollection` instance created
             to identify the quantiles values of each of the violin's
             distribution.
-
         """
 
         # Statistical quantities to be plotted on the violins
         means = []
         mins = []
         maxes = []
         medians = []
-        quantiles = np.asarray([])
+        quantiles = []
+
+        ppmins = []  # left end of quantiles lines
+        ppmaxs = []  # right end of quantiles lines
```

Comment:
Can we make better names? There's `ppmins` and `pmins`, but `ppmaxs` and `pmaxes`, and I don't even know what the `p` prefix means.

Reply:
sure